mlflow/langchain/chat_agent_langgraph.py

```diff
@@ -1,15 +1,18 @@
 import json
 import uuid
-from typing import Annotated, Any, Generator, Literal, Optional, TypedDict, Union
-
-from langchain_core.messages import AnyMessage, BaseMessage
-from langchain_core.messages import ToolCall as LCToolCall
-from langchain_core.runnables import RunnableConfig
-from langchain_core.runnables.utils import Input
-from langgraph.graph.state import CompiledStateGraph
-from langgraph.prebuilt.tool_node import ToolNode
-from langgraph.store.base import BaseStore
-from pydantic import BaseModel
+from typing import Annotated, Any, Generator, Optional, TypedDict
+
+try:
+    from langchain_core.messages import AnyMessage, BaseMessage, convert_to_messages
+    from langchain_core.runnables import RunnableConfig
+    from langchain_core.runnables.utils import Input
+    from langgraph.graph.state import CompiledStateGraph
+    from langgraph.prebuilt.tool_node import ToolNode
+except ImportError:
+    raise ImportError(
+        "Please install `langchain>=0.2.17` and `langgraph>=0.2.0` to use LangGraph ChatAgent"
+        "helpers."
+    )
```

Comment:
```suggestion
except ImportError as e:
    raise ImportError(
        "Please install `langchain>=0.2.17` and `langgraph>=0.2.0` to use LangGraph ChatAgent"
        "helpers."
    ) from e
```
so that we know which import fails